CONTRIBUTING.md

```diff
@@ -85,8 +89,24 @@ Note: Some tests will fail if `localhost` is not listed first in `/etc/hosts` fi
 
 For more information, including details on how WildFly Elytron is integrated in WildFly Core and WildFly, check out our [developer guide](https://wildfly-security.github.io/wildfly-elytron/getting-started-for-developers/).
 
+
 ## Contributing Guidelines
 
+### Testing
+
+Ensure that your changes are thoroughly tested before submitting a pull request. Follow these testing guidelines:
+
+- Run the existing unit tests using Maven: `mvn clean test`
+- Include new unit tests for your code changes.
+
+If applicable, provide instructions on how to run integration tests or any additional testing procedures.
+
+### Documentation
+Contributors are encouraged to keep documentation up-to-date along with code changes. If your changes impact user-facing features, update the relevant documentation files in directory.
```

Comment:
@harshittiwariii Just a minor, community documentation can be found in the wildfly repository https://github.com/wildfly/wildfly/tree/main/docs

Reply:
[ELY-2455]
@Skyllarr  Updated and requested for review.